roles/edpm_update/tasks/main.yml

```diff
@@ -19,12 +19,21 @@
 
 - name: Apply kernel patch via kpatch
   ansible.builtin.include_tasks: kpatch.yml
-  when: edpm_update_enable_kpatch
+  when:
+  - edpm_update_enable_kpatch
+  - not ansible_local.bootc
 
 - name: Update packages
   ansible.builtin.include_tasks: packages.yml
-  when: edpm_update_enable_packages_update
+  when:
+  - edpm_update_enable_packages_update
+  - not ansible_local.bootc
 
 - name: Update containers
   ansible.builtin.include_tasks: containers.yml
-  when: edpm_update_enable_containers_update
+  when:
+  - edpm_update_enable_containers_update
```

Comment:
@steveb There still updated here currently. Since we haven't landed on an implementation for quadlets yet, this change is just working within the existing framework.

I haven't submitted a PR for the quadlets work yet, but I'm trying to work on something that is consistent between RHEL and Image mode RHEL initially. Then we can build on that and assess the value of logically bound containers again. If we did want to pre-fetch containers as part of our image, we could still do that in the build process and it would work in the existing framework anyway.

Reply:
OK, sounds good to me